MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -358,6 +369,43 @@ class MultiPartyViewController: UIViewController {
     func updateLocalAudioState(hasAudio: Bool) {
         localParticipantAudioIndicator.isHidden = hasAudio;
     }
+
+    func updateLocalNetworkQualityLevel(networkQualityLevel: TVINetworkQualityLevel) {
+        logMessage(messageText: "Network Quality Level: \(networkQualityLevel.rawValue)")
+        var networkQualityLevelImage: UIImage?
+        var tintColor: UIColor?
+
+        switch networkQualityLevel {
+        case .unknown:
+            networkQualityLevelIndicator.isHidden = true
```

Comment:
should we set networkQualityLevelImage = nil and check it in logic at line 401 instead ?